doc_source/environment-variables.md

```diff
@@ -115,4 +115,37 @@ You can use the following environment variables that are accessible by default w
 |  \_BUILD\_TIMEOUT  |  The build timeout duration in minutes\.  |  30  | 
 
 **Note**  
-The `AMPLIFY_AMAZON_CLIENT_ID` and `AMPLIFY_AMAZON_CLIENT_SECRET` environment variables are OAuth tokens, not an AWS access key and secret key\. 
\ No newline at end of file
+The `AMPLIFY_AMAZON_CLIENT_ID` and `AMPLIFY_AMAZON_CLIENT_SECRET` environment variables are OAuth tokens, not an AWS access key and secret key\. 
+
+# Environment secrets<a name="environment-secrets"></a>
+
+Environment secrets are similar to environment variables, but they are SSM parameter store key value pairs and can be encrypted.
+
+## Set environment secrets<a name="setting-env-secrets"></a>
+
+**To set environment secrets**
+
+1. Sign in to the AWS Management Console and open the [AWS Systems Manager Console](https://console.aws.amazon.com/systems-manager/)\.
+
+1. In the AWS Systems Manager Console, choose Parameter Store, and then choose Create Parameter.
+
+1. Create a parameter name in the format `/amplify/{your_app_id}/{your_backend_environment_name}/{your_parameter_name}`
+
+1. Under type, choose Secure String for encrypted values
+
+1. Under KMS key source, choose my currenct account to use the default key for your account
+
+1. Under Value, enter the value you want
```

Comment:
**Value**

Reply:
Maybe it's just me, but I have no clue what value I'd add -- maybe an example?

Reply:
a private key perhaps.